test_nx/nx_g.e1

```diff
@@ -0,0 +1,2 @@
+src,dst
+1,3
```

Comment:
what's this for?

Reply:
This file is used for NX graph test, 
```
g4 = load_from(vertices={}, edges={}
            'v1': ([("id", VId.INT)], loader(vfile1)),
            'v2': ([("id", VId.INT)], loader(vfile2)),
            'v3': ([("id", VId.INT)], loader(vfile3)),
            'v4': ([("id", VId.INT)], loader(vfile4))
        },
        edges={
            'e1': ('v1', 'v2', 'both_out_in', [("src", SrcId.INT), ("dst", DstId.INT)], loader(efile)),
            'e2':('v3', 'v4', 'both_out_in', [("src", SrcId.INT),("dst", DstId.INT)],loader(efile))})
```
Therefore, there are four vertex files and two edge files.

Reply:
ignore this case, this case can use a property-graph.